### PR TITLE
Building and run unikernels with ops

### LINKED ISSUE #4390: Building and Run Unikernels with OPS
### Topic Suggestion
Building and Run Unikernels with OPS 
#### Proposed title of article
Building and Run Unikernels with OPS 

#### Proposed article introduction
Because of their size, security, and performance, unikernels have long been hailed as the next generation of cloud architecture, allowing megatrends like microservices and serverless to thrive while combatting the never-ending barrage of data breaches, cryptojacking, and other issues. However, there's a major issue. Developers have been slow to respond. Some outside the ecosystem have blamed bogus issues such as a lack of debugging, but this isn't the case. For example, gdb is frequently used by everyone I know that uses unikernels.

#### Key takeaways 
1 Introduction to OPS
2 Code, build and run Unkernel

#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Gladysoby**
#### Formatting and structure checks
- [X] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [X] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [X] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [X] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [X] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [X] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [X] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [X] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #4390

### CODE REVIEW

**Comment by Gladysoby** [comment]
@jumamiller this the article update.
have a look at it.
Thank you.

**Comment by ahmadmardeni1** [comment]
Please use the [new author template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/UPLOAD_INSTRUCTIONS.md).

File: content/articles/building-and-run-unikernel-with-ops/index.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+ 
+
```
Comment (jumamiller) [inline content/articles/building-and-run-unikernel-with-ops/index.md:2]: Work on these spacing, remove them!

File: content/articles/building-and-run-unikernel-with-ops/index.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
+ 
+
+### Building and Run Unikernels with OPS
+
+### Introduction
+
+Unikernels hаve lоng  been hаiled аs the next рhаse оf сlоud struсture due tо their length, seсurity, аnd рerfоrmаnсe, аllоwing megаtrends like miсrоserviсes аnd serverless tо рersist desрite the never-ending bаrrаge оf dаtа breасhes, сryрtоjасking, аnd оther рrоblеms. There could be a few issues as the construction industry has been sluggish in responding. Some atmospheres have blаmed fake troubles and a loss of debugging, but this isn't the case. Everyone I know who utilizes unikernels, for example, uses gdb regularly. OPS is the means used to build and run unikernels in any application they need and in any language.
```
Comment (jumamiller) [inline content/articles/building-and-run-unikernel-with-ops/index.md:7]: This sentence is very long and hard to read! Breakdown into paragraphs!

**Review by jumamiller — CHANGES_REQUESTED** [review]
You have very long sentences, kindly break them down!

**Comment by hectorkambow** [comment]
@Gladysoby 
Good afternoon and thank you for submitting your topic to the EngEd program.  It seems this PR is still quite a bit aways from being completed. 

>As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published.  All revisions and edits should be completed on your own forked repo (as to not build up a backlog in the queue). 

I'm seeing this is a duplicate pull request (#4857) (all pull request are looked at a case by case basis - but generally), any duplicate pull request should NOT require additional revisions - as they should have been caught by the author before re-submitting. 

We will be closing this for now. 

Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
 

👍 
 
cc @jumamiller

**Pull request closed without merging**